dags/hivemind_etl_helpers/tests/integration/test_discord_fetch_raw_messages.py

```
@@ -286,3 +286,177 @@
+            "isGeneratedByWebhook": False,
+        }
+        raw_data.append(data)
+
+        client[guild_id]["rawinfos"].insert_many(raw_data)
+
+        # Fetch messages from a specific date (October 3, 2023)
+        messages = fetch_raw_messages(guild_id, selected_channels=channels)
+        # Check if the fetched messages are equal to the expected messages
+        self.assertCountEqual(messages, 1)
+
+    def test_fetch_raw_messages_fetch_limited_characters(self):
```

Ruff (0.8.0)

Redefinition of unused `test_fetch_raw_messages_fetch_limited_characters` from line 290

(F811)